src/menu.py

```diff
@@ -0,0 +1,72 @@
+# menu.py
+# author: Daniel Jauergui
+# date: 3-31-2015
+
+import os
+from utils.singleton import Singleton
+
+class Menu(object):
+
+    __metaclass__ = Singleton
+
+    def __init__(self, name, items=None):
+        self.name = name
+        self.items = []
+
+    def add_item(self, item):
+        """Get a tuple of id,name and function, in order to get action for each
+        item added to current Menu
+
+        Keyword arguments:
+        item -- A Tuple of int, string object eg: (1,'Item of Menu', function).
+        """
+        self.items.append(item)
+
+    def clear_items(self):
+        """Remove all items of current menu.
+        """
+        self.items = []
+
+    def draw(self):
```

Comment:
draw is a very generic name, can we give something more especific according the context